components/director/internal/domain/runtime/service_test.go

```diff
@@ -245,7 +245,7 @@ func TestService_Create(t *testing.T) {
 			labelSvc := testCase.LabelUpsertServiceFn()
 			scenariosSvc := testCase.ScenariosServiceFn()
 			engineSvc := testCase.EngineServiceFn()
-			svc := runtime.NewService(repo, nil, scenariosSvc, labelSvc, idSvc, engineSvc)
+			svc := runtime.NewService(repo, nil, scenariosSvc, labelSvc, idSvc, engineSvc, ".*_defaultEventing$")
```

Comment:
Maybe we could add tests for creating a runtime with provided protected labels and see that they are being ignored.

Reply:
I agree - sounds like a realistic case